internal/seeder/seed.go

```diff
@@ -59,18 +61,23 @@ func (s *SeedService) Start(ctx context.Context) error {
 func (s *SeedService) insertFromIterator(it objectIterator) error {
 	for {
 		obj, err := it.Next()
-		if err == iterator.Done {
-			break
-		}
-
 		if err != nil {
-			return err
+			if err == iterator.Done {
+				break
+			}
+
+			return fmt.Errorf("Error retrieving iterator object: %v", err)
```

Comment:
https://go.dev/wiki/CodeReviewComments#error-strings, error message should not be capitalized.